parsers/NA.py

```diff
@@ -107,6 +130,7 @@ def fetch_production(zone_key = 'NA', session=None, target_datetime=None, logger
                                    logger=logger)
 
     production = data_processor(raw_text)
+    check_timestamp = timestamp_checker(session=session, logger=logger)
```

Comment:
probably just
`timestamp_checker(session=session, logger=logger)`
since there is no return value